orchestra/todos/api.py

```diff
@@ -30,32 +40,45 @@ def add_todolist_template(todolist_template_slug, task_id):
             get_cond_props = locate(path)
             cond_props = get_cond_props(task.project)
         except Exception:
-            logger.exception('Invalid condition function path.')
+            logger.exception('Invalid conditional function path.')
     for template_todo in template_todos:
         _add_template_todo(
             template_todo, todolist_template, root_todo, task, cond_props)
 
 
-def _to_exclude(props, conditions=[]):
+def _to_exclude(props, conditions):
+    """
+    The conditions is it a list of conditions that get ORed together,
+    with properties in each dictionary getting ANDed.
+    """
+    any_condition_true = False
+
     for condition in conditions:
+        all_props_true = True
```

Comment:
are we fine with a zero-item list of properties evaluating to true? could do `all_props_true = True and len(all_props_true) > 0`